### PR TITLE
tgui-next portable atmos

### PR DESCRIPTION
**Pull request by actioninja**
## About The Pull Request

Also a tweak to titlecase what's in a titlebar.

image
image

## Changelog
:cl:
add: tgui-next portable atmos
/:cl:

### CODE REVIEW

File: tgui-next/packages/tgui/components/TitleBar.js [hunk 1]
```diff
@@ -27,7 +28,7 @@ export const TitleBar = props => {
         color={statusToColor(status)}
         name="eye" />
       <div className="TitleBar__title">
-        {title}
+        {toTitleCase(title)}
```
Comment (stylemistake) [inline tgui-next/packages/tgui/components/TitleBar.js]: Have you checked if this plays nicely with things like "PanD.E.M.I.C"?

**Review by stylemistake — COMMENTED** [review]
👍

File: tgui-next/packages/tgui/components/TitleBar.js (hunk 1 above)
Comment (actioninja) [reply tgui-next/packages/tgui/components/TitleBar.js]: I'll have to check. Probably would make the most sense to not titlecase it if it already contains caps.

Comment (stylemistake) [reply tgui-next/packages/tgui/components/TitleBar.js]: Or huge refactoring.

**Review by stylemistake — DISMISSED** [review]

File: tgui-next/packages/tgui/components/TitleBar.js (hunk 1 above)
Comment (actioninja) [reply tgui-next/packages/tgui/components/TitleBar.js]: Fixed now.

**Pull request merged**